.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: csharp
+sudo: false
+mono:
+  - latest
+
+install:
+  - nuget install Cake -Version 0.6.4 -OutputDirectory tools/
```

Comment:
Looks like it is from mono 3.8.0. Was going to ask about this once I'd got it up and running - the main project runs on both mono-latest and mono-3.2.8 specifically. Is there a reason for that?